### PR TITLE
Unused code samples are removed

### LINKED ISSUE #2424: Remove unused code samples
The following code samples are no longer used in the docs:

- [ ] `faceted_search_2 `

Remove them from the following files:
- `.code-samples.meilisearch.yaml`
- `sample-template.yaml`

### PR DESCRIPTION
**Pull request by roy9495**
# Pull Request

## Related issue
Fixes #2424 

## What does this PR do?
-Removes the unused code sample of faceted_search_2  from these two docs .code-samples.meilisearch.yaml and sample-template.yaml

## PR checklist
Please check if your PR fulfils the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -498,15 +498,6 @@ getting_started_update_ranking_rules: |-
       "release_date:asc",
       "rank:desc"
     ]'
-getting_started_update_displayed_attributes: |-
```
Comment (brunoocasali) [inline .code-samples.meilisearch.yaml:501]: @maryamsulemani97, but this is present in the 👀  https://github.com/meilisearch/documentation/blob/b838b99e6b26ee900295ea2d06144e899cbeddbf/learn/meilisearch_101/customizing_relevancy.mdx?plain=1#L42

**Review by brunoocasali — COMMENTED** [review]
Thanks a lot, @roy9495, for your contribution!! 🎉 

I have a question for @maryamsulemani97 just checking if the issue was up to date! besides that looks good to me!

File: .code-samples.meilisearch.yaml (hunk 1 above)
Comment (maryamsulemani97) [reply .code-samples.meilisearch.yaml:501]: You're right @brunoocasali! Sorry about that 🙏

**Review by maryamsulemani97 — CHANGES_REQUESTED** [review]
Hi @roy9495!

Thank you for the PR! As pointed out by @brunoocasali, we are using `getting_started_update_displayed_attributes` in the docs. Could you please add it back to both files? 🙏  I'll update the issue as well!

**Review by maryamsulemani97 — APPROVED** [review]
LGTM! 🚀 
Thanks again @roy9495!

**Pull request merged**